doc/website/getting-started/examples/intermediate-examples.md

```diff
@@ -0,0 +1,5 @@
+# Intermediate examples
+
+{! ./../iceoryx_examples/waitset/README.md !}
+{! ./../iceoryx_examples/waitset_on_c/README.md !}
+{! ./../iceoryx_examples/singleprocess/README.md !}
```

Comment:
After a discussion with @MatthiasKillat we decided that the ice_multi_publisher example doesn't teach the users anything additionally. It will most likely be removed. To showcase the n:m communication all icedelivery and icedelivery_in_c binaries will be started in an additional iceensemble example